ArjunaJTS/integration/src/main/java/com/arjuna/ats/jbossatx/jts/TransactionManagerService.java

```diff
@@ -92,7 +100,7 @@ public void start(org.omg.CORBA.ORB theCorbaORB) throws Exception
     public void stop() {
         super.stop();
 
-        ORB.getInstance("jboss-atx").shutdown();
+        ORB.getInstance(ORB_NAME).shutdown();
```

Comment:
Thanks Mike, that makes perfect sense now. I will mark it for merge but acknowledge CI is broken